apple/LayoutReanimation/REAAnimationsManager.m

```diff
@@ -454,6 +461,16 @@ - (BOOL)startAnimationsRecursive:(REAUIView *)view
     _startAnimationForTag(view.reactTag, EXITING, preparedValues);
   }
 
+  // NOTE: even though this view is still visible,
+  // since it's removed from the React tree, we won't
+  // start new animations for it, and might as well remove
+  // the layout animation config now
+  _clearAnimationConfigForTag(view.reactTag);
+
+  if (!wantAnimateExit) {
```

Comment:
Why did we move this logic in the code?

Reply:
I wanted to call the cleanup function before `return`, so the tag gets removed form `shouldAnimateExitingForTag_`. On android it was already implemented this way, so I decided to change it here.